helm-chart/values.yaml

```diff
@@ -1,12 +1,12 @@
 # NGINX Service Mesh image registry settings.
 registry:
   # Hostname:port (if needed) for registry and path to images. 
-  # Affects: nginx-mesh-api, nginx-mesh-metrics, nginx-mesh-sidecar, nginx-mesh-init, nginx-mesh-cert-reloader
+  # Affects: nginx-mesh-api, nginx-mesh-cert-reloader, nginx-mesh-init, nginx-mesh-metrics, nginx-mesh-sidecar
```

Comment:
The csi driver is missing from this list and the imageTag list

Reply:
No helm support for OpenShift yet.

Reply:
Ah, right.